ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -613,6 +613,11 @@ tests:
   steps:
     test:
     - ref: stackrox-e2e
+- as: merge-gke-version-compatibility-tests
```

Comment:
1. `gke-version-compatibility-tests` above has timeout 5 hours, but here `merge-gke-version-compatibility-tests` does not have it. Does it make sense to add it for consistency? Is 5 hours timeout relevant at all?
https://github.com/openshift/release/pull/34109/files#diff-73b7e51a14ffbb458892e5f811bf393e3c334ad8003aab64d8bea1f05880b169R342-R347
2. Also, could you please for consistency move `gke-version-compatibility-tests` (lines 342-347) to be between `gke-upgrade-tests` and `gke-postgres-qa-e2e-tests`?

Reply:
1. I don't think it's relevant (at least tests never timed out so far) but adding it can't hurt so I'll do it.
2. Will do

Reply:
Just double checked and none of the `merge-` tests have a timeout even when the non `merge-` versions do so it's more consistent not to add one now.